src/Components/button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import "./button.css"
+
+function Button({type, className, clickHandler, children, name}) {
+    return (
+        <button
+            type={type}
+            onClick={clickHandler}
+            className={className}
+            name={name}>
+            {children}
+        </button>
+    );
+}
```

Comment:
Dit component is nu een doorgeefluik. Alle properties die binnenkomen worden direct doorgezet op één component. Je kan dus net zo goed direct het html button element gebruiken